debian/additions/mariadb.conf.d/50-server.cnf

```diff
@@ -50,8 +51,12 @@ bind-address            = 127.0.0.1
 #general_log_file       = /var/log/mysql/mysql.log
 #general_log            = 1
 
+# Error logging goes to syslog due to /etc/mysql/conf.d/mysqld_safe_syslog.cnf
```

Comment:
and systemd goes to stdout/stderr and therefore into journalctl. I was going to suggest commenting this out. Its a common pain point of supporting users that have `systemctl status mariadb.service` showing none of the mysqld error log that could actually help, and then starts the process of finding it.

Reply:
log_error is already commented out and I now extended the inline comment to explain the points you mentioned here